devnet-sdk/contracts/constants/constants.go

```diff
@@ -6,10 +6,43 @@ import (
 )
 
 var (
-	L2ToL2CrossDomainMessenger types.Address = common.HexToAddress("0x4200000000000000000000000000000000000023")
-	SuperchainWETH             types.Address = common.HexToAddress("0x4200000000000000000000000000000000000024")
-	ETHLiquidity               types.Address = common.HexToAddress("0x4200000000000000000000000000000000000025")
-	SuperchainTokenBridge      types.Address = common.HexToAddress("0x4200000000000000000000000000000000000028")
+	L2ToL1MessagePasser           types.Address = common.HexToAddress("0x4200000000000000000000000000000000000016")
```

Comment:
nit: maybe sort at least the 0x42... entries for easy lookup?

Reply:
done!